src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx

```diff
@@ -0,0 +1,188 @@
+import { EntityHeader, Sans, Spacer } from "@artsy/palette"
+import { RecommendedArtist_artist } from "__generated__/RecommendedArtist_artist.graphql"
+import { track } from "Artsy/Analytics"
+import * as Schema from "Artsy/Analytics/Schema"
+import { SystemContext } from "Artsy/SystemContext"
+import { FillwidthItem } from "Components/Artwork/FillwidthItem"
+import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "Components/FollowButton/FollowArtistButton"
+import { Carousel } from "Components/v2"
+import { stringify } from "qs"
+import React, { FC, useContext } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { data as sd } from "sharify"
+import { get } from "Utils/get"
+
+interface RecommendedArtistProps {
+  artist: RecommendedArtist_artist
+}
+const HEIGHT = 150
+
+// TODO: Remaining tasks
+/*
+ * find out tracking constants to use
+ * Move handleOpenAuth/openMobileAuth/openDesktopAuth to shared location
+ * Buy now/make offer labels
```

Comment:
It looks like the 'badges' have only been added to `GridItem`: https://github.com/artsy/reaction/blob/686719aa1c8594a8a6baa73312e7c13c77b2c7ca/src/Components/Artwork/GridItem.tsx#L183 , so perhaps the work item is to extract that out into its own component and include that in `FillwidthItem` as well?